test/run_core_test.exs

```diff
@@ -185,9 +185,9 @@ defmodule EwebmachineTest do
         allowed_methods do: ["POST"]
         post_is_create do: true
 	# Check modified state is propagated
-        defh create_path, do: {state.path, conn, state}
+        defh create_path(conn, %{path: path} = state), do: {path, conn, state}
         content_types_accepted do: ["text/plain": :from_text]
-        defh from_text, do: {true, conn, state |> Map.put(:path, "titus")}
+        defh from_text(conn, state), do: {true, conn, state |> Map.put(:path, "titus")}
```

Comment:
I feel like there are several places in documentation where we could steer toward this more explicit usage (e.g. https://github.com/kbrw/ewebmachine/blob/master/lib/ewebmachine/builder.handlers.ex#L131) if we wanted to.

Do you think we should ? I'm usually keener on it, but you guys are probably more aware of the EWM usage than me.